### DISCUSSION TITLE
Are Pinball Machine IDs unique to Pinballmap?

### DISCUSSION THREAD

**Discussion by ApplezJuice** [pinballmap/pbm#1580] [Q&A] [answered]
The return from a GET that includes a machine ID, is this a unique ID that is stored on pinball map or is this the machine id from say ipdb? I am fairly certain it is just an ID unique to pinball map but just making sure! :) Thanks for this awesome API by the way! In the brainstorm and design process for an idea and will be using the pinball map API most likely.

**Comment by RyanTG** [comment, marked as answer]
Yes, the ID is just our database ID. But in the machines table we also store the IPDB id (when available) and the OPDB id (when available).

**Reply by ApplezJuice** [reply]
Awesome ty
